python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -92,7 +94,8 @@ def initLog(cls, longlog):
             pythonLogger.addHandler(cls._lsstLogHandler)
         else:
             cls._recordComponentSetting(None)
-            logging.basicConfig(level=logging.INFO)
+            logging.basicConfig(level=logging.INFO,
+                                **(dict(format=cls.pylog_longLogFmt if longlog else dict())))
```

Comment:
The alternative with an `if`/`else` statement is only two lines longer and may be clearer.

Reply:
@ktlim I'm not sure I really like any of the ways I've tried optionally passing a variable (or not). Can you show me the code you're thinking of?

Reply:
Something simple like this:
```
if longlog:
    logging.basicConfig(level=logging.INFO)
else:
    logging.basicConfig(level=logging.INFO, format=cls.pylog_longLogFmt)
```